### DISCUSSION TITLE
Quorum transactions are directed to contract of address ‘0x00000….’

### DISCUSSION THREAD

**Discussion by archm49** [ConsenSys/quorum#1497] [Q&A]
#### System information

Geth version: 1.9.7-stable
OS: Amazon Linux 2
commit hash: 8027cec5

#### Expected behaviour
Transaction directed to correct contract address

#### Actual behaviour
Transaction directed to ‘0x00000….’ contract address

#### Backtrace

````
[backtrace]
````

When submitting logs: please submit them as text and not screenshots.

<img width="1071" alt="Screenshot 2022-09-07 at 10 42 31 AM" src="https://user-images.githubusercontent.com/99325132/188833114-4d5e7f59-ae02-48e3-a496-fc1c4884aad8.png">

**Comment by antonydenyer** [comment]
We need more information than this. Can you repoduce the problem using [dev-quickstart](https://github.com/ConsenSys/quorum-dev-quickstart)?
